src/uwtools/cli/run_forecast.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=duplicate-code
+# pylint: disable=duplicate-code, unused-import
```

Comment:
What's the unused import? I haven't found a reason yet to use this particular disable directive instead of removing the actually unused import. Maybe TIL something?

Reply:
Ohh - ok, this was a weird one, might be an issue of the implementation. I'm catching `UWError` on line 36 instead of `UWConfigError` like previously. However, I was mocking `UWConfigError` in `test_run_forecast()` to test catching the base from the parent, but it was erroring on not having `UWConfigError`. If I include it without disabling the import, it doesn't count it below and errors. Strictly, its a test remnant for an issue I hadn't resolved to where I was happy with it